content/docs/introduction/comparison.md

```diff
@@ -132,14 +132,15 @@ boundaries like products, services, datacenters, or similar aspects.
 Independent servers (which can be run redundantly in parallel) may also give
 you better reliability and failure isolation.
 
-Kapacitor currently has no [built-in distributed/redundant
+Kapacitor's open-source release currently has no [built-in distributed/redundant
```

Comment:
I'd like to keep the same style as above, where the open source options are compared first and then the commercial option after all that.